scripts/manual_operations/manual_remove.py

```diff
@@ -91,6 +91,8 @@ def multiple_versions_found(self, run_number):
         # Remove runs that the user does NOT want to delete from the delete list
         for reduction_job in self.to_delete[run_number]:
             if not int(reduction_job.run_version) in user_input:
+                # this removes the versions that the user DID NOT enter
+                # from the list of run versions that will be DELETED
                 self.to_delete[run_number].remove(reduction_job)
```

Comment:
```suggestion
        self.to_delete[run_number] = [
            reduction_job for reduction_job in self.to_delete[run_number]
            if reduction_job.run_version in user_input
        ]
```
Maybe we should change this to remove the double negation, and instead make to_delete contain only the run_versions that the user has specified